GitUp/Application/PreferencesWindowController.h

```diff
@@ -0,0 +1,35 @@
+//
+//  PreferencesWindowController.h
+//  Application
+//
+//  Created by Dmitry Lobanov on 08.10.2019.
+//
+
+#import <AppKit/AppKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+#pragma mark - Release Channels
+extern NSString* const PreferencesWindowController_ReleaseChannel_Stable;
+extern NSString* const PreferencesWindowController_ReleaseChannel_Continuous;
+
+#pragma mark - Themes
+extern NSString* const PreferencesWindowController_Theme_SystemPreference;
+extern NSString* const PreferencesWindowController_Theme_Light;
+extern NSString* const PreferencesWindowController_Theme_Dark;
+
+@interface PreferencesThemeService : NSObject
++ (NSString *)selectedTheme;
++ (void)applySelectedTheme;
+@end
+
+@interface PreferencesWindowController : NSWindowController
+@property (nonatomic, copy) NSArray <NSString *>*channelTitles;
+@property (nonatomic, copy) NSArray <NSString *>*themesTitles;
```

Comment:
Nit: Generics right next to type: `NSArray<` (no space)

Reply:
```objective-c
@property (nonatomic, copy) NSArray<NSString *>*themesTitles;

// or

@property (nonatomic, copy) NSArray<NSString*>*themesTitles;

// or 

@property (nonatomic, copy) NSArray<NSString*> *themesTitles;
```

Reply:
The style throughout the rest of the project is to have the type all in one blob and then have the identifier after a space so my vote would be `NSArray<NSString*>* propertyName`. Not how my personal projects would go, but it's not really my repo 😅.

Reply:
@lucasderraugh Fixed. :3